queue/abstract.lua

```diff
@@ -574,6 +586,24 @@ local function identification_init()
     end
 end
 
+--- Create an expiration fiber to cleanup expired sessions.
+local function create_expiration_fiber()
+    local exp_fiber = fiber.new(function()
```

Comment:
Just noted: fiber.new() is supported since 1.10.0-27-g48cad35eb.

Reply:
Done.